.github/workflows/needs-attention.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - uses: actions/stale@5bef64f19d7facfb25b37b414482c7164d639639 # v9.1.0
         with:
-          days-before-stale: 30
+          days-before-stale: 90
```

Comment:
I'm open to changing this back to 30, I wanted to focus on reversing the mechanism - all things need attention (triage) until they have been reviewed.